lib/mutant/env/bootstrap.rb

```diff
@@ -137,7 +137,7 @@ def expression(scope)
         name = scope_name(scope) or return
 
         unless name.instance_of?(String)
-          warn("#{scope.class}#name from: #{scope.inspect} returned #{name.inspect}. #{SEMANTICS_MESSAGE}")
+          warn("#{scope.class}#name from: #{scope} returned #{name}. #{SEMANTICS_MESSAGE}")
```

Comment:
since this is duplicated

`warn("#{scope.class}#name from: #{scope} ` + ` #{SEMANTICS_MESSAGE}")`

it might read better if you had a `semantic_warning` method